### ISSUE TITLE
Cmd+k not working on mac in screen reader mode

### ISSUE THREAD

**Issue by eleanorjboyd** [microsoft/vscode#177901]
Testing #177735

While testing the TPI I was unable to complete this step: `The hover is keyboard accessible via ctrl+k, ctrl+i (or cmd+k, cmd+i) when either the terminal or the terminal tab is focused`. The steps I tried were
1. Going into screen reader optimized mode on vscode
2. trying all 4 options ( ctrl+k, ctrl+i (or cmd+k, cmd+i) ) but cmd+k should be the right one

**Comment by legomushroom** [comment]
Does not reproduce on stable 1.95.3(f1a4fb) on Mac OSx 15.1.1 Sequoia so closing as fixed (the `cmd-` one works, `ctrl-` one does not).
